### PR TITLE
docs: add js numeric precision problem in faq

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

docs

#### What this PR does / why we need it (en: English/zh: Chinese):

en: add js numeric precision problem in faq
zh: 在 faq 中新增解决 JS 精度问题的方法

#### Which issue(s) this PR fixes:

None.

### CODE REVIEW

File: content/zh/docs/hertz/faq/_index.md [hunk 1]
```diff
@@ -61,3 +61,90 @@ type HandlerFunc func(c context.Context, ctx *RequestContext)
...
+
+    var u User
+    u.ID = 6855337641038665531
+    j, err := json.Marshal(&u)
+    if err != nil {
+        panic(err)
+    }
+
+    h.GET("/hello", func(ctx context.Context, c *app.RequestContext) {
+        c.String(consts.StatusOK, string(j))
```
Comment (li-jin-gou) [inline content/zh/docs/hertz/faq/_index.md:109]: why not use c.Json()

File: content/en/docs/hertz/faq/_index.md [hunk 2]
```diff
@@ -67,3 +67,85 @@ The `ctx` is primarily used to store request-level variables, which are recycled
...
+func main() {
+    h := server.Default()
+    
+    h.GET("/hello", func(ctx context.Context, c *app.RequestContext) {
+        var u User
+        err := json.Unmarshal([]byte(`{"id":6855337641038665531}`), &u)
+        if err != nil {
+            panic(err)
+        }
+        c.JSON(consts.StatusOK, utils.H{
```
Comment (li-jin-gou) [inline content/en/docs/hertz/faq/_index.md:144]: use struct instead of utils.H{

Comment (L2ncE) [reply content/en/docs/hertz/faq/_index.md]: > use struct instead of utils.H{

done, thx

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
LGTM

**Pull request merged**